File: app/src/main/java/com/kota/Telnet/TelnetConnectorListener.java
package com.kota.Telnet;

public interface TelnetConnectorListener {
    void onTelnetConnectorClosed(TelnetConnector telnetConnector);

    void onTelnetConnectorConnectFail(TelnetConnector telnetConnector);

    void onTelnetConnectorConnectStart(TelnetConnector telnetConnector);

    void onTelnetConnectorConnectSuccess(TelnetConnector telnetConnector);

    void onTelnetConnectorReceiveDataFinished(TelnetConnector telnetConnector);

    void onTelnetConnectorReceiveDataStart(TelnetConnector telnetConnector);
}

File: app/src/main/java/com/kota/telnet/TelnetConnectorListener.kt
package com.kota.telnet

interface TelnetConnectorListener {
    fun onTelnetConnectorClosed(telnetConnector: TelnetConnector?)

    fun onTelnetConnectorConnectFail(telnetConnector: TelnetConnector?)

    fun onTelnetConnectorConnectStart(telnetConnector: TelnetConnector?)

    fun onTelnetConnectorConnectSuccess(telnetConnector: TelnetConnector?)

    fun onTelnetConnectorReceiveDataFinished(telnetConnector: TelnetConnector?)

    fun onTelnetConnectorReceiveDataStart(telnetConnector: TelnetConnector?)
}
